test/EntityFramework6.Npgsql.Tests/Support/EntityFrameworkTestBase.cs

```diff
@@ -213,6 +224,87 @@ private static DbCompiledModel CreateModel(NpgsqlConnection connection)
                 null);
             dbModel.StoreModel.AddItem(echoFunc);
 
+            var stringStoreType = dbModel.ProviderManifest.GetStoreTypes().First(x => x.ClrEquivalentType == typeof(string));
+            var modelBlogStoreType = dbModel.StoreModel.EntityTypes.First(x => x.Name == typeof(Blog).Name) as EntityType;
```

Comment:
Nit: "as EntityType" is redundant and generates a useless warning, you can safely remove it.